src/index.js

```diff
@@ -78,7 +78,6 @@ class Server extends EventEmitter {
             this.conf,
             this.logger.push(LOG_ID.INBOUND),
             this.cache,
-            this.metricsClient,
```

Comment:
Is it ok to remove this? What was it doing?

Reply:
that's a copy-paste type